Branch-SDK/src/io/branch/indexing/ContentDiscoverer.java

```diff
@@ -165,7 +165,12 @@ public void run() {
 
                         // Cache the analytics data for future use
                         PrefHelper.getInstance(activity).saveBranchAnalyticsData(contentEvent_);
-                        lastActivityReference_ = null;
+
+                        if (cdManifest_.getDiscoveryRepeatTime() != -1) {
```

Comment:
Can we set the "drt" threshold to 500 in order to avoid  any smaller value of DRT kill the app